pyproject.toml

```diff
@@ -19,6 +19,7 @@ dependencies = [
   # Browser Automation: Locked to specific versions to avoid breaking changes.
   "patchright==1.52.4",
   "requests>=2.32.4",
+  "python-fasthtml>=0.12.21",
```

Comment:
This should be in `dev` of `dependency-groups`, as we need it only for ACME tests and not the main server.